docs/users-and-roles-swagger.yaml

```diff
@@ -164,7 +164,32 @@ paths:
         responses: {}
         httpMethod: POST
         type: aws_proxy
-
+  /users/mine/accepted-terms:
+    put:
+      summary: Updates Terms and agreement for an existing User (person)
+      operationId: updateUserTermsConditions
+      security:
+        - CognitoUserPool: [ "https://api.nva.unit.no/scopes/backend", "https://api.nva.unit.no/scopes/frontend", "aws.cognito.signin.user.admin" ]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UpsertTermsConditionsRequest'
+      responses:
+        '202':
+          description: Successful request for updating the user. User will be eventually updated
+          headers:
+            Location:
+              schema:
+                type: string
+              description: Relative path to the user URI
```

Comment:
I see that following swagger file misses a lot of respons codes for multiple endpoints. I think you should add more possible responses here, like 401? 404? 502?